google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClientImpl.java

```diff
@@ -112,6 +126,16 @@ public Timestamp writeAtLeastOnce(final Iterable<Mutation> mutations) throws Spa
   public CommitResponse writeAtLeastOnceWithOptions(
       final Iterable<Mutation> mutations, final TransactionOption... options)
       throws SpannerException {
+    if (useMultiplexedSessionForBlindWrite) {
+      return getMultiplexedSession().writeAtLeastOnceWithOptions(mutations, options);
```

Comment:
I think that this will also use regular sessions if multiplexed sessions for read-only has not been enabled. Meaning that in the current setup:
1. It is impossible to enable multiplexed sessions for only blind writes.
2. You have to enable multiplexed sessions for read-only transactions in order to be able to enable if for blind writes.

See also my comment above; If possible, I would prefer just one setting that enables multiplexed sessions for everything that supports it.

Reply:
Have removed the flag.